cli/src/command.rs

```diff
@@ -327,24 +331,37 @@ impl<O: DescriptorOpts> Exec for Args<Command, O> {
             }
             Command::Construct {
                 v2,
-                invoice,
+                to: beneficiaries,
                 fee,
                 psbt: psbt_file,
             } => {
                 let mut runtime = self.bp_runtime::<O::Descr>(&config)?;
-                // TODO: Support lock time and RBFs
-                let params = TxParams::with(*fee);
+
                 // Do coin selection
-                let coins: Vec<_> = match invoice.amount {
-                    Amount::Fixed(sats) => {
+                let total_amount =
+                    beneficiaries.iter().try_fold(Sats::ZERO, |sats, b| match b.amount {
+                        Amount::Max => Err(()),
+                        Amount::Fixed(s) => sats.checked_add(s).ok_or(()),
+                    });
+                let coins: Vec<_> = match total_amount {
+                    Ok(sats) if sats > Sats::ZERO => {
                         runtime.wallet().coinselect(sats, coinselect::all).collect()
                     }
-                    Amount::Max => {
+                    _ => {
+                        eprintln!(
+                            "Warning: you are not paying to anybody but just aggregating all your \
+                             balances to a single UTXO",
+                        );
                         runtime.wallet().all_utxos().map(WalletUtxo::into_outpoint).collect()
                     }
                 };
-                let psbt = runtime.wallet_mut().construct_psbt(&coins, *invoice, params)?;
+
+                // TODO: Support lock time and RBFs
```

Comment:
RBF is a feature that's core to how we implement RGB in BitMask, for Transfer Batching... Not sure if this affects our capability to batch transfers through RBF?

Reply:
I was assuming these days you do not need to opt-in for RBF, since I am always able to get my  non-RBF transactions RBFed due to the fact that most nodes by default are now `--mempoolfullrbf`

Reply:
Let's please not depend on that

Reply:
If it helps, we can default to RBF on... It's better for anonymity also, since most P2TR tx have RBF enabled